unit-test-security-rules-v9/test/database.spec.mjs

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import assert from 'assert';
+import { readFileSync, createWriteStream } from 'fs';
+import http from 'http';
+
+// Workarounds for npm not recognizing named exports in cjs.
```

Comment:
Does `const { ... } = require()` syntax work?

Reply:
Works fine in `cjs` files, not with `mjs` and native imports